README.md

```diff
@@ -1,18 +1,15 @@
-![Brainly Style Guide](https://i.imgur.com/mnmDILk.png)
-
-[![Build Status](https://travis-ci.org/brainly/style-guide.svg)](https://travis-ci.org/brainly/style-guide)
-[![Dependency Status](https://david-dm.org/brainly/style-guide.svg)](https://david-dm.org/brainly/style-guide)
-[![devDependency Status](https://david-dm.org/brainly/style-guide/dev-status.svg)](https://david-dm.org/brainly/style-guide#info=devDependencies)
+![Brainly Style Guide](cover.png)
 
 This project is a living style guide of all basic front-end components used at Brainly.
 
 ## Documentation
 
 Documentation is an esential part of this project. You can see its latest version on [styleguide.brainly.com](https://styleguide.brainly.com). New version of the docs is deployed with every merge to master. You can also browse older versions by changing version number in the URL. Documentation is splitted into four sections:
-  * Basics - colors, fonts, icons, simple components
-  * Components - complex components
-  * Containers - components that host other components
-  * Interactive - new, interactive version of the style guide docs that we are experimenting with
+
+- Basics - colors, fonts, icons, simple components
+- Components - complex components
+- Containers - components that host other components
+- Interactive - new, interactive version of the style guide docs that we are experimenting with
```

Comment:
Maybe `- Interactive - interactive version of the style guide docs`? I guess it is nothing new anymore 😄